sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -187,6 +187,16 @@ static TestEnvironment()
         /// </summary>
         public string ClientId => GetRecordedVariable("CLIENT_ID");
 
+        /// <summary>
+        ///   The storage container endpoint to use during Import Live tests. Recorded.
+        /// </summary>
+        public string StorageContainerEndpoint => GetRecordedVariable("STORAGE_CONTAINER_URI");
```

Comment:
These seem like they are specific to your service, so they should go in the service level TestEnvironment class.

Reply:
Good point - moved to sdk/digitaltwins/Azure.DigitalTwins.Core/tests/DigitalTwinsTestEnvironment.cs. Thanks!